plugins/main/loopback/loopback.go

```diff
@@ -49,6 +50,36 @@ func parseNetConf(bytes []byte) (*types.NetConf, error) {
 	return conf, nil
 }
 
+func GetRootNetNS() (netns.NsHandle, error) {
+	return netns.GetFromPid(1)
+}
+
+func IsRootNetNS(nsPath string) (bool, error) {
+	ns, err := netns.GetFromPath(nsPath)
+	if err != nil {
+		return false, err
+	}
+	defer func() {
+		err = ns.Close()
+		if err != nil {
```

Comment:
Since the program is going away, you can ignore the result from `ns.Close()`

Reply:
In other words, `defer ns.Close()`

Reply:
Thanks for your advice. Done